libpromises/evalfunction.c

```diff
@@ -6634,68 +6633,103 @@ static FnCallResult FnCallReadRealList(EvalContext *ctx, ARG_UNUSED const Policy
     return ReadList(ctx, fp, args, CF_DATA_TYPE_REAL);
 }
 
+static FnCallResult _ReadDataGeneric(const char *const fname,
```

Comment:
I think we've mentioned this before, but: We don't use `_` for internal / static functions.